Doc/library/asyncio-eventloop.rst

```diff
@@ -188,21 +188,25 @@ Running and stopping the loop
 .. coroutinemethod:: loop.shutdown_default_executor(timeout=None)
 
    Schedule the closure of the default executor and wait for it to join all of
-   the threads in the :class:`ThreadPoolExecutor`. After calling this method,
+   the threads in the :class:`~concurrent.futures.ThreadPoolExecutor`.
+   After calling this method,
    :meth:`loop.run_in_executor` will raise a :exc:`RuntimeError` if called
    while using the default executor.
 
-   The *timeout* parameter specifies the amount of time this method gives the
-   executor to finish joining. The default value is ``None``, which means the
+   *timeout* specifies the duration in seconds (as a :class:`float`)
```

Comment:
Call me old-fashioned, but I don't like starting a sentence with a lowercase letter, even if it's part of an identifier, and I don't like capitalizing the name either, so I liked the original phrasing:
```suggestion
   The *timeout* parameter specifies the duration in seconds (as a :class:`float`)
```

Reply:
Yeah, I don't like it either, but on the other hand it was more concise and direct, as appropriate for an API reference (and per Diataxis). It just comes down which you want to prioritize...

Reply:
@bskinn can choose then.